src/cmd.rs

```diff
@@ -24,6 +24,12 @@ pub enum Op {
     Set(f32),
 }
 
+#[derive(Clone, Debug, PartialEq)]
+pub enum Flip {
+	Horizontal,
+	Vertical,
+}
```

Comment:
Let's call this `Axis`, there's a few other places we could re-use this type that way.

Reply:
Yea, I was not sure about what to call it, `Axis` sounds much better. In regards to it being reusable, is `src/cmd.rs` the correct place for this type?

Reply:
this should be resolved now